src/chapters/lists/lists-are-like-strings.rst

```diff
@@ -31,11 +29,6 @@ number of elements in the list).
    In line 3, ``len(letters)`` returns the number of items stored in the
    ``letters`` list.
 
-.. admonition:: Note
-
-   We will explore how the ``len()`` function deals with nested lists (like
-   ``[['a', 'b', 'c'], ['x', 'y', 'z']]``) later in this chapter.
-
 Note that the statement ``print(letters[len(letters)])`` will throw an *index
 out of range* error. Since index values start at 0, the last element in any
 list will always have a value of ``len(list_name) - 1``.
```

Comment:
I don't think so! I removed the note because the students are probably not even thinking about nested lists at this point in the chapter so drawing attention to it could be more distracting than helpful.

Reply:
I agree with removing the note!  I was asking about the following paragraph that talks about the index out of range error.

Originally, I had it in a Note admonition, but it looked funny following a different Note, so I formatted it as a normal paragraph.  Now that we've removed the first box, should we put "Note that the statement ``print(letters[len(letters)])`` will..." back in an admonition?